README.md

```diff
@@ -20,143 +20,20 @@ _Thanks to the [Better Jinja](https://github.com/samuelcolvin/jinjahtml-vscode)
 - Block comment toggling: Press `CMD + /` to create HubL comments
 - Block level indentation: Text inside of `{% block %}` will be indented automatically
 
-### __HubL Snippets__
-All HubL supported tags, filters, expression tests and functions have auto-complete snippets. Expression tests are accessed by typing the test name alone, filters are accessed with `|` and functions/tags are accessed with `~`. All snippets include descriptions and parameter details. You up/down arrow to navigate the IntelliSense and hit enter to execute a snippet. Snippet completed HubL statements will auto-highlight available parameters, which can be tabbed through (`${parameter}`).
+### HubL Snippets
+All HubL supported tags, filters, expression tests and functions have auto-complete snippets. Expression tests are accessed by typing the test name alone, filters are accessed with `|` and functions/tags are accessed with `~`. All snippets include descriptions and parameter details. You up/down arrow to navigate the IntelliSense and hit enter to execute a snippet. Snippet completed HubL statements will auto-highlight available parameters, which can be tabbed through (`${parameter}`). Additionally there is support for other commonly needed syntax, such as module fields.
```

Comment:
This section becomes a little hard to read. Maybe after first sentence bullet how to access each auto completion?

> A prefix is required to access some auto completions:
> 
> - Expression tests are accessed by typing the test name alone.
> - filters are accessed with `|`.
> - functions/tags are accessed with `~`.

Also noticed typo. "You"

>All snippets include descriptions and parameter details. Use `up`/`down` arrow keys to navigate the [IntelliSense](https://code.visualstudio.com/docs/editor/intellisense) and hit `enter` to execute a snippet.